src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/integration_tests/tests/web/IndexPageTests.kt

```diff
@@ -0,0 +1,58 @@
+package org.vaccineimpact.orderlyweb.tests.integration_tests.tests.web
+
+import org.assertj.core.api.Assertions
+import org.jooq.impl.DSL
+import org.jsoup.Jsoup
+import org.junit.Test
+import org.vaccineimpact.orderlyweb.db.JooqContext
+import org.vaccineimpact.orderlyweb.db.Tables.*
+import org.vaccineimpact.orderlyweb.models.Scope
+import org.vaccineimpact.orderlyweb.models.permissions.ReifiedPermission
+import org.vaccineimpact.orderlyweb.tests.integration_tests.tests.IntegrationTest
+
+class IndexPageTests : IntegrationTest()
```

Comment:
I think changing these to integration tests should fix the issue of sometimes not finding a completely unpublished report among the demo data, since integration tests get a new db every test so it shouldn't matter how publish-happy the other tests get...